src/services/codefixes/importFixes.ts

```diff
@@ -435,6 +437,23 @@ namespace ts.codefix {
                     return position;
                 }
 
+                function getSingleQuoteStyleFromExistingImports() {
+                    const firstModuleSpecifier = forEach(sourceFile.statements, node => {
+                        switch (node.kind) {
+                            case SyntaxKind.ImportDeclaration:
+                                return (<ImportDeclaration>node).moduleSpecifier;
+                            case SyntaxKind.ImportEqualsDeclaration:
+                                const moduleReference = (<ImportEqualsDeclaration>node).moduleReference;
+                                return moduleReference.kind === SyntaxKind.ExternalModuleReference ? moduleReference.expression : undefined;
+                            case SyntaxKind.ExportDeclaration:
+                                return (<ExportDeclaration>node).moduleSpecifier;
+                        }
+                    });
+                    if (firstModuleSpecifier && isStringLiteral(firstModuleSpecifier)) {
+                        return sourceFile.text.charCodeAt(skipTrivia(sourceFile.text, firstModuleSpecifier.pos)) === CharacterCodes.singleQuote;

```

Comment:
Is `skipTrivia(sourceFile.text, firstModuleSpecifier.pos)` the same as `firstModuleSpecifier.getStart()`?